pages/base_page.py

```diff
@@ -173,10 +171,6 @@ def report_list(self):
             select = Select(report_dropdown)
             return [opt.text for opt in select.options]
 
-        @property
-        def is_exploitable_crash_report_present(self):
-            return self.is_element_present(*self._exploitable_crash_report_locator)
-
```

Comment:
It turns out the tests had some duplication, these two lines were essentially doing the same thing.

    assert 'Exploitable Crashes' not in csp.header.report_list
    assert csp.header.is_exploitable_crash_report_present is not True

The 'is_exploitable_crash_report_present' method is quite fragile. I decided to remove it in favor of the `csp.header.report_list` assertion method.